src/control/nav2_dwa_controller/include/nav2_dwa_controller/common_types.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
Not sure how we can have common types across projects unless there is a shared (sub-) project that they both depend on.

Reply:
You are right. The changes under dwa_controller are half-baked. I'll remove them for this PR.